gammapy/cube/new.py

```diff
@@ -235,25 +236,31 @@ def make_map_hadron_acceptance(pointing, livetime, bkg, ref_geom, offset_max):
     background : `~gammapy.maps.WcsNDMap`
         Background predicted counts sky cube in reco energy
     """
-    # Compute offsets of all pixels
-    offset = make_separation_map(ref_geom, pointing).quantity
-
-    # Retrieve energies from WcsNDMap
-    # Note this would require a log_center from the geometry
-    energy = ref_geom.axes[0].center * ref_geom.axes[0].unit
 
     # Compute the expected background
     # TODO: properly transform FOV to sky coordinates
     # For now we assume the background is radially symmetric
 
     # TODO: add a uniform API to the two background classes
+    energy_axis = ref_geom.axes[0]
+    # Compute offsets of all pixels
+    offset = make_separation_map(ref_geom, pointing).quantity
```

Comment:
@registerrier 
This is a bit uggly and due to exposure at the end of the method put at 0 for offset>offset_max